src/loc/LocPublishLinkButton.tsx

```diff
@@ -23,10 +22,8 @@ export default function LocPublishLinkButton(props: Props) {
             locItem={ props.locItem }
             publishMutator={ async (current, callback) => {
                 if(current instanceof EditableRequest) {
-                    return await publishLink({
-                        locState: current,
+                    return current.legalOfficer.publishLink({
```

Comment:
Link is published only on `OpenLoc`, not on `EditableRequest` - See [my comment](https://github.com/logion-network/logion-api/pull/165#discussion_r1238179276) in the SDK PR.

Reply:
Indeed, if you don't mind, I'll fix this in an upcoming PR with the updated dependency.